salt/orchestrate/edx/services/elasticsearch.sls

```diff
@@ -84,7 +87,7 @@ build_dogwood_elasticsearch_nodes:
 
 remove_broken_line_from_elasticsearch_init_script:
   salt.function:
-    - tgt: 'G@roles:elasticsearch and G@environment:dogwood-rp'
+    - tgt: 'G@roles:elasticsearch and G@environment:dogwood-qa'
```

Comment:
That's because this orchestration file is for the instances in the QA environment and they were accidentally targeting RP instances which means that it wasn't performing those last couple of steps on the right instances.